packages/eas-cli/src/commands/update/list.ts

```diff
@@ -0,0 +1,135 @@
+import { getConfig } from '@expo/config';
+import { Flags } from '@oclif/core';
+import chalk from 'chalk';
+import CliTable from 'cli-table3';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import { ViewAllUpdatesQuery } from '../../graphql/generated';
+import { UpdateQuery } from '../../graphql/queries/UpdateQuery';
+import Log from '../../log';
+import { findProjectRootAsync, getProjectIdAsync } from '../../project/projectUtils';
+import { promptAsync } from '../../prompts';
+import { FormatUpdateParameter, UPDATE_COLUMNS, formatUpdate } from '../../update/utils';
+import groupBy from '../../utils/expodash/groupBy';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+import { getVcsClient } from '../../vcs';
+
+type UpdateGroupDescription = FormatUpdateParameter & {
+  branch: string;
+  group: string;
+  platforms: string;
+  runtimeVersion: string;
+};
+
+export default class BranchView extends EasCommand {
+  static description = 'View the recent updates for a branch';
+
+  static flags = {
+    branch: Flags.string({
+      description: 'List all updates on this branch',
+      exclusive: ['all'],
+    }),
+    all: Flags.boolean({
+      description: 'List all updates associated with this project',
+      exclusive: ['branch'],
+      default: false,
+    }),
+    json: Flags.boolean({
+      description: `Return a json with all of the recent update groups.`,
+      default: false,
+    }),
+  };
+
+  async runAsync(): Promise<void> {
+    let {
```

Comment:
```suggestion
    const {
```
Don't use let unless you have a really good reason.